webscraping/courseSequences/scraping/scraper.js

```diff
@@ -81,7 +96,7 @@ let scrapeAllUrls = (function (){
                 console.log("\n** Scraping weirdly-formatted sequences from elecCoenHeadings.json **\n");
                 startElecCoenScrape();
             }
-        };
+        }; 
```

Comment:
quick space

Reply:
extremely slick, extremely quick, so quick I didnt' even see it zoom by my screen. Mahd deuces in your tramzone good sir